### PR TITLE
refactor(cookies): remove ConfigLoader::areCookiesEnabled

### PR DESCRIPTION
**Pull request by philbooth**
@shane-tomlinson

### CODE REVIEW

**Comment by TDA** [comment]
Question to Shane: Should the files still be named cookies-disabled when we arent checking for cookies inside them?
Other than that, LGTM! :+1: We'll wait for Shane to press the magic green button :dancer: 
@shane-tomlinson

**Comment by philbooth** [comment]
@saipc: just regarding the view name, I figured that would fall out naturally as part of #1140.

For now, the error page we display talks about cookies so it seems right that the view is named for cookies too.

A discrepancy exists between the check we perform and the info that is presented to the user. When we fix that discrepancy, the view name can change too (I figured).

**Comment by TDA** [comment]
@philbooth Sounds good! :+1:

File: app/scripts/views/cookies_disabled.js [hunk 1]
```diff
@@ -18,26 +19,22 @@ function (Cocktail, BaseView, ConfigLoader, AuthErrors, Template, BackMixin) {
       BaseView.call(this, options);
 
       this._configLoader = new ConfigLoader();
+      this._storage = options.storage || Storage;
```
Comment (shane-tomlinson) [inline app/scripts/views/cookies_disabled.js]: Can you capitalize the `S` in both `this._storage` and `options.storage`? The lowercase convention indicates an instance, here we are using the module itself.

**Comment by shane-tomlinson** [comment]
@philbooth - aside from my `s`=>`S` nit, looks great, works well & tests all pass! Merge when updated.

**Pull request merged**